assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -96,8 +95,11 @@ describe( 'modules/analytics properties', () => {
 					{ body: response, status: 500 }
 				);
 
-				muteConsole( 'error' );
-				await registry.dispatch( STORE_NAME ).createProperty( accountID );
+				registry.dispatch( STORE_NAME ).createProperty( accountID );
+
+				await subscribeUntil( registry,
+					() => registry.select( STORE_NAME ).isDoingCreateProperty( accountID ) === false
+				);
```

Comment:
The previous version was correct here – async actions should simply be `await`-ed rather than using `subscribeUntil` as they return a Promise which resolves when the action is finished. Generally speaking, `subscribeUntil` is a low-level utility that shouldn't be needed in most tests. It's mostly used now in older tests where we didn't have an alternative yet.